libs/core/src/lib/multi-input/multi-input.component.html

```diff
@@ -1,61 +1,54 @@
+                    <fd-tokenizer [compact]="compact" class="fd-multi-input-tokenizer">
+                        <fd-token
+                            *ngFor="let token of selected"
+                            [disabled]=this.disabled
+                            (onCloseClick)="handleSelect(false, token)">
+                            <span [innerHtml]="token | displayFnPipe:displayFn"></span>
+                        </fd-token>
+                        <input type="text" class="fd-input fd-tokenizer__input fd-multi-input-tokenizer-input"
+                               fd-form-control fd-input-group-input
+                               #searchInputElement
+                               [placeholder]="placeholder"
+                               [(ngModel)]="searchTerm"
+                               (ngModelChange)="handleSearchTermChange()"
+                               (keydown)="handleInputKeydown($event)">
+                    </fd-tokenizer>
                 </fd-input-group>
-            </div>
-        </fd-popover-control>
-        <fd-popover-body [attr.aria-hidden]="!open">
-            <fd-menu class="fd-multi-input-menu-overflow"
-                     *ngIf="displayedValues && displayedValues.length"
-                     [style.maxHeight]="maxHeight">
-                <ul fd-menu-list>
-                    <li *ngFor="let value of displayedValues; let ind  = index">
-                        <label fd-menu-item (keydown)="handleKeyDown($event, ind)">
-                            <fd-checkbox [ngModel]="selected ? selected.indexOf(value) !== -1 : false"
-                                         (ngModelChange)="handleSelect($event, value)">
-                                <span [innerHtml]="value | displayFnPipe:displayFn | highlight:searchTerm:highlight"></span>
-                            </fd-checkbox>
-                        </label>
-                    </li>
-                </ul>
-            </fd-menu>
-            <ng-content></ng-content>
-        </fd-popover-body>
-    </fd-popover>
-</div>
-<div class="fd-multi-input-tags">
-    <fd-token
-            *ngFor="let token of selected"
-            [disabled]=this.disabled
-            (onCloseClick)="handleSelect(false, token)"
-            class="fd-multi-input-token-spacing">
-        <span [innerHtml]="token | displayFnPipe:displayFn"></span>
-    </fd-token>
+            </fd-popover-control>
+            <fd-popover-body [attr.aria-hidden]="!open">
+                <fd-menu class="fd-multi-input-menu-overflow"
+                         *ngIf="displayedValues && displayedValues.length"
+                         [style.maxHeight]="maxHeight">
+                    <ul fd-menu-list>
+                        <li *ngFor="let value of displayedValues; let ind  = index">
+                            <label fd-menu-item (keydown)="handleKeyDown($event, ind)">
+                                <fd-checkbox [ngModel]="selected ? selected.indexOf(value) !== -1 : false"
```

Comment:
There is a problem with this particular implementation. If you click inside the `label` inside the `li` element in popover (outside of checkbox label but still inside of the list element) checkbox is going to be selected but only visually! Checkbox value is updated on checkbox label click, I guess it'a a problem with checkbox.

Reply:
What browser are you seeing this with?  I fixed it (with Chrome at least), do you have the latest?